CONTRIBUTING.adoc

```diff
@@ -0,0 +1,46 @@
+= Contributing
+// settings:
+:idprefix:
+:idseparator: -
+:source-language: ruby
+:language: {source-language}
```

Comment:
`language` and `source-language` don't seem to be used. Maybe we should remove them.